Explorer/Assets/Scripts/ECS/StreamableLoading/Common/Systems/LoadSystemBase.cs

```diff
@@ -108,7 +108,7 @@ private void Execute(in Entity entity, ref StreamableLoadingState state, ref TIn
 
                 // if the request is cached wait for it
                 // If there is an ongoing request it means that the result is neither cached, nor failed
-                if (cache.OngoingRequests.SyncTryGetValue(intention.CommonArguments.URL, out UniTaskCompletionSource<StreamableLoadingResult<TAsset>?> cachedSource))
+                if (cache.OngoingRequests.SyncTryGetValue(intention.CommonArguments.URL.GetCacheableURL(), out var cachedSource))
```

Comment:
Can we allocate it only once and pass it as an argument?
In the current code, we execute regex and allocate every time.

Reply:
Of course, makes sense. Introduced the cacheable string as a field